README.md

```diff
@@ -217,112 +222,113 @@ created during these steps with your initial SNS developer neurons).
    [Test canister](https://github.com/dfinity/sns-testing#test-canister)
    for further details) which can be thought of as a placeholder
    for your dapp.
-2. Run the script
+
+2. Give control of your dapp canisters to NNS:
    ```bash
-   ./deploy_sns.sh <config-path>  # from Bash
+   ./let_nns_control_dapp.sh  # from Bash
    ```` 
-   to deploy an SNS passing the path to the SNS configuration file as an argument.
-   A sample configuration is available in the file `./sns-test.yml`.
-3. Run the script
+   This automatically creates an SNS
+   configuration file named `sns_init.yaml`, unless such a file already exists
+   (e.g. you hand-crafted one yourself). The auto-generated file assumes that
+   you are using the test dapp.
+
+3. Submit an NNS proposal to create an SNS that will control your dapp canister(s):
    ```bash
-   ./register_dapp.sh <canister-id>  # from Bash
+   ./propose_sns.sh  # from Bash
    ``` 
-   to register the canister with the provided canister ID with the SNS deployed in the previous step.
-   If you deployed the example dapp provided with this repo, you can run
-   ```bash
-   ./bin/dfx canister id test  # from Bash
-   ```
-   to get the canister id.
-   After this step, the SNS is able to manage the canister.
+   The proposal will pass right away, because it is made by a neuron that has an
+   overwhelming amount of voting power (this is part of the testing environment).
 
-4. (Optionally) upgrade your dapp by submitting an SNS proposal that can be voted on using the SNS developer neuron.
+   After a few minutes, you should see a new SNS instance in the [Launchpad]
+   section of the NNS dapp.
 
-   The purpose of this step is to check that your dapp could be upgraded after the SNS is already created but before the swap begins, for example, to issue a hotfix.
+   [Launchpad]: http://qsgjb-riaaa-aaaaa-aaaga-cai.localhost:8080/launchpad
 
-   This step requires your dapp repo to have an upgrade script that interacts with the replica via the 8080 port.
+4. Optional: Upgrade your dapp canister(s) via SNS proposal.
 
-   If you don't yet have a solution to upgrade your custom dapp, you can still proceed with these instructions by upgrading the example dapp using the scripts provided with this repo:
+   If you are going through these instructions using the test dapp, this step 
+   can be done as follows:
 
    ```bash
-   ./upgrade_test_canister.sh  # from Bash
```

Comment:
why drop `#from Bash` (also elsewhere)

Reply:
This is just a copy-paste error. Thanks

Reply:
Please resolve once you fixed all occurrences.